library/core/src/cmp.rs

```diff
@@ -790,6 +790,7 @@ pub trait Ord: Eq + PartialOrd<Self> {
     /// ```
     #[must_use]
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_diagnostic_item = "ord_cmp_method"]
```

Comment:
Why is this prefixed with `ord_`? most other methods/associated functions in this PR seem to not be prefixed by the trait...

Reply:
The other traits all have methods named after the trait.